client/client-lib/Cargo.toml

```diff
@@ -1,10 +1,14 @@
 [package]
 name = "mint-client"
-version = "0.1.0"
-authors = ["The Fedimint Developers"]
-edition = "2021"
 description = "mint-client provides a library for sending transactions to the federation."
-license = "MIT"
+version.workspace = true
```

Comment:
Wait, what? We need to do it this way for every field in every package? That's so meh...

Reply:
Looks like it's true... :shrug:

Reply:
yeah.. would be nice if you could inherit a whole group or smth.. still better like this for changing stuff in the future